tests/component/sections/shared/pagination/PaginationResultsInfo.spec.tsx

```diff
@@ -53,4 +53,15 @@ describe('PaginationResultsInfo', () => {
     )
     cy.findByText('10 of 15 Items seen').should('exist')
   })
-})
+
+  it('shows the correct formatted results when accumulated prop passed and results are more than page size', () => {
+    cy.customMount(
+      <PaginationResultsInfo
+        paginationInfo={new PaginationInfo<DatasetPaginationInfo>(1, 100, 1500)}
+        accumulated={500}
+      />
+    )
+    cy.findByText('500 of 1,500 Items seen').should('exist')
```

Comment:
Maybe we could format again the expected total, so we don't hardcode the translated total.
Something like:
```
const formattedTotalCount = new Intl.NumberFormat().format(1500)

cy.findByText(`500 of ${formattedTotalCount} Items seen`).should('exist')
```

Reply:
Made a slight tweak and added two variables: `expectedFormattedTotalCount` and `totalCount` to help with the hardcoding.